### ISSUE TITLE
Integrate with Firebase

### ISSUE THREAD

**Issue by agentmilindu** [CodeLanka/ez-net-app#3]
Integrate with Firebase to fetch data from Firebase.

**Comment by nushydude** [comment]
Is Firebase going to be our data store?

**Comment by agentmilindu** [comment]
Yes, exactly.

**Comment by nushydude** [comment]
If you I a project in the Firebase console when you are logged into your Google account, that project will be created under my account, wouldn't it? Don't we need to create an account for the project? Sorry I haven't really worked on a React-Native project before, so I don't know how this works.

**Comment by agentmilindu** [comment]
Yeah, that's right, I already asked @tdevinda to create a Firebase project under GDG Sri Lanka Firebase account yesterday. I hope he saw the message.

**Comment by nushydude** [comment]
I'll wait for his input. Until then I'll work on refactoring the code.

**Comment by agentmilindu** [comment]
I added Firebase files, now we can work on this.

**Comment by nushydude** [comment]
What exactly have you added? I cannot seem to find them. 

What should be the next step?

**Comment by agentmilindu** [comment]
Check this PR https://github.com/CodeLanka/ez-net-app/pull/17

**Comment by nushydude** [comment]
But we still don't have a firebase account,  right?

**Comment by agentmilindu** [comment]
We have! :D Check `google-services.json`

```
"project_info": {
    "project_number": "502584752547",
    "firebase_url": "https://project-7593821828655173684.firebaseio.com",
    "project_id": "project-7593821828655173684",
    "storage_bucket": "project-7593821828655173684.appspot.com"
  }
```

**Comment by ManuelJF** [comment]
@pamuditha need help to integrate firebase sdk???
